packages/object-utilities/src/js/__protected.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Creates an protected property in the given object
+ * Use this for protected properties, functions are public for now.
+ *
+ * First Usage:
+ * __protected(this, {foo:"bar"})
+ *
+ * After that:
+ * read: var foo = __protected(this).foo;
+ * write: __protected(this).foo = "baz";
+ *
+ * @param {Object} object – given object
+ * @param {Object} [value] – given value
+ * @return {Object} protected property / context
+ */
+export default function __protected(object, value) {
```

Comment:
yeah, the name isnt perfect, but sadly "protected" is a reserved word and we wanted to make sure it is a "system utility". if you have a better idea - please tell us :)